package.json

```diff
@@ -26,9 +27,11 @@
     "typescript": "^4.3.5"
   },
   "dependencies": {
+    "@kubernetes/client-node": "^0.15.1",
```

Comment:
This adds lots of memory fat.. reason why previous api calls were handcrafted. Would it be possible to implement autoupdate without this library?

Reply:
Nevermind, it's only imported in the updater so should not affect agent memory consumption, right?.

Reply:
I agree, shouldn't have an effect there.